react-rendering-lists/my-app/src/RenderList.jsx

```diff
@@ -0,0 +1,19 @@
+const pokedex = [
+  { number: '001', name: 'Bulbasaur' },
+  { number: '004', name: 'Charmander' },
+  { number: '007', name: 'Squirtle' },
+  { number: '025', name: 'Pikachu' },
+  { number: '039', name: 'Jigglypuff' },
+];
+
+export default function RenderList() {
+  const listItems = pokedex.map((obj) => <li key={obj.name}>{obj.name}</li>);
+
+  return (
+    <>
```

Comment:
These empty `<>` are not needed here since you only have one top-level element (`<ul>`).